k8s/aditum/chart/aditum/templates/application.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: "{{ .Release.Name }}"
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+    app.kubernetes.io/name: "{{ .Release.Name }}"
+  annotations:
+    kubernetes-engine.cloud.google.com/icon: >-
+      data:image/png;base64,{{ .Files.Get "logo.png" | b64enc }}
+    marketplace.cloud.google.com/deploy-info: '{"partner_id": "click-to-deploy-containers", "product_id": "aditum", "partner_name": "Click To Deploy"}'
+spec:
+  descriptor:
+    type: Custom Governance
+    version: "{{ .Values.aditum.image.tag }}"
+  info:
+    - name: Custom Governance site URL
+      value: "https://{{ .Values.certificate.hostname }}"
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: "{{ .Release.Name }}"
+  componentKinds:
+  - group: cloud.google.com/v1beta1
```

Comment:
Please remove versions from `group`s. For core resources such as `Secret`, the value for group should be an empty string `group: ''`.